README.md

```diff
@@ -103,6 +110,15 @@ Swagger documentation is visiable on allowed hostes, Update environement variabl
 
 Code released under the [MIT License](https://opensource.org/licenses/MIT).
 
+This software uses libraries from the [FFmpeg](https://www.ffmpeg.org) project.
```

Comment:
Probably need to mention that we use only non-GPL code from ffmpeg. Also mention that exact steps to compile the project can be found in dockerfile...